src/Components/Shifting/CommentsSection.tsx

```diff
@@ -9,10 +9,68 @@ import { IComment } from "../Resource/models";
 import PaginatedList from "../../CAREUI/misc/PaginatedList";
 import request from "../../Utils/request/request";
 
-interface CommentSectionProps {
+interface CommentProps {
   id: string;
+  comment: string;
+  created_by_object: any;
```

Comment:
Can we have proper types here? We have a pre-defined type: "PerformedByUserModel" which can be used here.

Reply:
The 'PerformedByUserModel' interface does not have the required type that I need for the 'commentSection', so I can't use it here. Therefore, I defined a new interface.
export type UserBareMinimum = {
  id: number;
  username: string;
  first_name: string;
  last_name: string;
  email: string;
  user_type: UserRole;
  last_login: string | undefined;
};


interface CommentProps {
  id: string;
  comment: string;
  created_by_object: any;
  modified_date: string;
}